### PR TITLE
😡 Use threads in meme channels

### PR DESCRIPTION
**Pull request by stepech**

### CODE REVIEW

**Comment by jirkavrba** [comment]
🧵

**Comment by dev-null-undefined** [comment]
proste tam napis at se to prijme

**Comment by dev-null-undefined** [comment]
Tohle by se mělo okamžitě mergnout!

File: src/HonzaBotner.Discord.Services/EventHandlers/MemeChannelMessageHandler.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        _memeChannels = options.Value.MemeChannels ?? Array.Empty<ulong>();
+        _client = wrapper.Client;
+    }
+
+    public async Task<EventHandlerResult> Handle(MessageCreateEventArgs args)
+    {
+        if (!args.Author.IsBot
+            && _memeChannels.Contains(args.Channel.Id)
+            && !args.Message.Attachments.Any()
+            && !args.Message.Content.Contains("https://"))
```
Comment (SoptikHa2) [inline src/HonzaBotner.Discord.Services/EventHandlers/MemeChannelMessageHandler.cs:30]: A co http? :slightly_frowning_face:

**Comment by stepech** [comment]
Local peasants are rebelling once more, this needs immediate merge to put them in their place!

**Pull request closed without merging**